versionBump.js

```diff
@@ -1,28 +0,0 @@
-const replace = require('replace-in-file');
-const pjVersion = require('./package.json').version;
-const fs = require('fs');
-
-const options = [
-  {
-    files: 'storybook-config.json',
```

Comment:
will need to figure out a way to update the storybook-config on build and deploy so that the versions addon works

Reply:
I think this is more of a job for the ci/cd flow.